performance_tests/results/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
I'm surprised this directory needs an __init__.py. What's going in this directory? .txt files? Do you actually need this?

Reply:
All the results go in this directory. Can we push a directory to github without any file inside?

Reply:
Ah. The convention is to put an empty `.gitkeep` file in the directory.

Reply:
Okay, done. Thanks.